packages/components/src/Calendar/CalendarRange/CalendarRange.module.scss

```diff
@@ -43,6 +43,7 @@ $cell-border-radius: 3px;
   justify-content: space-between;
   width: 100%;
   color: $color-purple-800;
+  top: -0.25rem;
```

Comment:
This is needed as the original rdp nav would actually jump out of its containing block.

v8 on the left and this branch on the right, this fixes that. Due to how the markup has changed the cleanest way was to pull the nav outside of it's containing block again.

![Screenshot 2025-05-14 at 3 43 06 pm](https://github.com/user-attachments/assets/6c703493-7cd2-46fc-b07b-7e32b9a5618d)